velox/core/QueryConfig.h

```diff
@@ -433,6 +440,11 @@ class QueryConfig {
     return get<std::string>(kSpillCompressionKind, "none");
   }
 
+  uint64_t spillWriteBufferSize() const {
+    // The default write buffer size set to 4MB.
```

Comment:
Why is the default set to 4MB? The PR description mentions an experiment that used 1MB.

Reply:
Yeah, it should be 1MB by default. Will tune this later after fix the memory efficiency issue in StreamArena